docs/tactics.md

```diff
@@ -1374,3 +1374,17 @@ See also additional documentation of `using_well_founded` in
 * `@[simps]` reduces let-expressions where necessary.
 * If one of the fields is a partially applied constructor, we will eta-expand it
   (this likely never happens).
+
+## simp_rw
+
+`simp_rw` is a version of `simp` which performs rewriting in the given order.
```

Comment:
I like the description from the module docstring (as a mix of simp and rw) better.  If you tell me it's a version of `simp`, then I expect it to 1) use simp lemmas and 2) have a similar syntax to `simp`.  Neither of which is true.